CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [v0.16.0](https://github.com/fastly/go-fastly/releases/tag/v0.16.0) (2020-06-25)
```

Comment:
I'm not sure why it did this. The command that I ran was:

```bash
bash -c 'CHANGELOG_GITHUB_TOKEN=$(cat ~/.github/credentials) SEMVER_TAG="v1.16.1" make changelog'
```

Even when I re-run it, it says no diff. I'm assuming this is an issue with the erroneous release that I recently created.

Reply:
Even when I delete `v0.16.0` from my remote, the changelog fails to generate anything.

I don't expect the changelog to have enhancements or anything, just link to Full Changelog (i.e., the list of commits).

Reply:
@phamann Thoughts on deleting the erroneous `v0.16.0` tag and associated release?

Reply:
Yep, think we should delete the 0.16.0 tag and release 👍